react/week2/my-app/src/Components/TodoItem.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+
+class TodoItem extends Component {
+  constructor() {
+    super();
+
+    this.onMarkItemComplete = (event) => {
+      this.props.markItemComplete(this.props.id);
+    };
+
+    this.onDeleteItem = (event) => {
+      this.props.deleteItem(this.props.id);
+    };
+  }
```

Comment:
Normally you only use a `constructor` when you need to manipulate the props that are normally in it's parameters.
So normally if you would use the `constructor` you should declare like this 👇 

`constructor(props) {
  super(props);
  ....
}`